Pipfile.lock

```diff
@@ -904,11 +874,8 @@
             "version": "==3.7.0"
         },
         "pytest-operator": {
-            "hashes": [
-                "sha256:be6c279e1c3a5c1fb281a5895efa7e22a56da6948f68b6f645a7b55548a1af49"
-            ],
-            "index": "pypi",
-            "version": "==0.11.0"
+            "git": "https://github.com/charmed-kubernetes/pytest-operator.git",
+            "ref": "efb6a1ee9522ff55dd120cdd4925d696c90ebff9"
```

Comment:
just gonna point out this jumps pytest-operator from 0.11.0 -> multimodel.  
There could be more than just these changes in it
* https://github.com/charmed-kubernetes/pytest-operator/pull/69